crates/bevy_ecs_compile_fail_tests/tests/ui/query_exact_sized_iterator_safety.stderr

```diff
@@ -0,0 +1,50 @@
+error[E0599]: the method `len` exists for struct `QueryIter<'_, '_, &Foo, ReadFetch<'_, Foo>, bevy_ecs::query::Changed<Foo>>`, but its trait bounds were not satisfied
+   --> tests/ui/query_exact_sized_iterator_safety.rs:8:33
+    |
+8   |       println!("{}", query.iter().len())
+    |                                   ^^^ method cannot be called on `QueryIter<'_, '_, &Foo, ReadFetch<'_, Foo>, bevy_ecs::query::Changed<Foo>>` due to unsatisfied trait bounds
+    |
+   ::: /home/runner/work/bevy/bevy/crates/bevy_ecs/src/query/iter.rs:16:1
```

Comment:
Okay, before I try resolving the error once again, I want to point out this part in`query_exact_sized_iterator_safety.stderr`.

In the error, it shows the exact file path where this file is located. As such, even though this can resolved by copy pasting the result from the CI logs, this will throw an error on anyone working on bevy that do not have their own local repository located exactly where it is stated here.

I will try to rewrite the test in such a way where it does not show the file path. Does anyone have a suggestion on how to do this? Thank you.